cmd/localCI/pullRequest.go

```diff
@@ -75,11 +75,13 @@ type PullRequest struct {
 // canBeTested returns an error if the pull request cannot be tested
 func (pr *PullRequest) canBeTested() error {
 	if !pr.Mergeable {
+		ciLog.Debugf("the pull request is not mergeable")
 		return fmt.Errorf("the pull request is not mergeable")
```

Comment:
@grahamwhaley  you can assign the error string first  to an error variable and use that for printing the debug message to avoid the repitition. Something like
err := fmt.Errorf("...")
ciLog.Debugf(err)
return err